components/Onboard/api/setCompletedOnboardingApi.ts

```diff
@@ -0,0 +1,19 @@
+import { Helpers } from "@quantfive/js-web-config";
+import API, { generateApiUrl } from "~/config/api";
+import { captureEvent } from "~/config/utils/events";
+
+export const setCompletedOnboardingApi = async (): Promise<void> => {
```

Comment:
Would the user see onboarding form again if they just don't click continue or skip but rather just visit the homepage? Ideally they should not

Reply:
Good point, updated it to auto-mark onboarding as complete if the user navigates away.